src/core_plugins/kibana/public/dashboard/__tests__/panel.js

```diff
@@ -14,7 +15,13 @@ describe('dashboard panel', function () {
 
   function init(mockDocResponse) {
     ngMock.module('kibana');
-    ngMock.inject(($rootScope, $compile, esAdmin) => {
+    ngMock.inject(($rootScope, $compile, Private, esAdmin) => {
+      Private.swap(SavedObjectsClientProvider, () => {
+        return {
+          get: sinon.stub().returns(Promise.resolve(mockDocResponse))
+        };
+      });
+
       sinon.stub(esAdmin, 'mget').returns(Promise.resolve({ docs: [ mockDocResponse ] }));
```

Comment:
Good catch - I was able to remove this esAdmin stub, but still some work to be done to remove the remaining fieldMapping call.